include/oembed.php

```diff
@@ -209,25 +209,32 @@ function oembed_format_object($j){
 	return mb_convert_encoding($ret, 'HTML-ENTITIES', mb_detect_encoding($ret));
 }
 
-function oembed_iframe($src,$width,$height) {
-
-	if(! $width || strstr($width,'%'))
-		$width = '640';
-	if(! $height || strstr($height,'%')) {
-		$height = '300';
-		$resize = 'onload="resizeIframe(this);"';
-	} else
-		$resize = '';
-
-	// try and leave some room for the description line.
-	$height = intval($height) + 80;
-	$width  = intval($width) + 40;
+/**
+ * Generates the iframe HTML for an oembed attachment. Width and height are given
```

Comment:
I also know this problem :-D At least not before drinking two cups of coffee ;-) Oh today I had only one.